javascript/index.js

```diff
@@ -3,38 +3,149 @@
+
+obtainInstruction("steak", 0)
+  .then((step0) => {
+    document.querySelector("#steak").innerHTML += `<li>${step0}</li>`;
+    return obtainInstruction("steak", 1);
+  })
+  //  ... Your code here
+  .then((step1) => {
+    document.querySelector("#steak").innerHTML += `<li>${step1}</li>`;
+    return obtainInstruction("steak", 2);
+  })
+  .then((step2) => {
+    document.querySelector("#steak").innerHTML += `<li>${step2}</li>`;
+    return obtainInstruction("steak", 3);
+  })
+  .then((step3) => {
+    document.querySelector("#steak").innerHTML += `<li>${step3}</li>`;
+    return obtainInstruction("steak", 4);
+  })
+  .then((step4) => {
+    document.querySelector("#steak").innerHTML += `<li>${step4}</li>`;
+    return obtainInstruction("steak", 5);
+  })
+  .then((step5) => {
+    document.querySelector("#steak").innerHTML += `<li>${step5}</li>`;
+    return obtainInstruction("steak", 6);
+  })
+  .then((step6) => {
+    document.querySelector("#steak").innerHTML += `<li>${step6}</li>`;
+    return obtainInstruction("steak", 7);
+  })
+  .then((step7) => {
+    document.querySelector("#steak").innerHTML += `<li>${step7}</li>`;
+    document.getElementById("steakImg").removeAttribute("hidden");
+  })
+  .catch((error) => console.log(error));
+
 // Iteration 3 using async/await
 // ...
+broccoli.push("Broccoli is ready!");
+
+async function makeBroccoli() {
+  // ... Your code here
+  try {
+    let broccoli0 = await obtainInstruction("broccoli", 0);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli0}</li>`;
+
+    let broccoli1 = await obtainInstruction("broccoli", 1);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli1}</li>`;
+
+    let broccoli2 = await obtainInstruction("broccoli", 2);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli2}</li>`;
+
+    let broccoli3 = await obtainInstruction("broccoli", 3);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli3}</li>`;
+
+    let broccoli4 = await obtainInstruction("broccoli", 4);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli4}</li>`;
+
+    let broccoli5 = await obtainInstruction("broccoli", 5);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli5}</li>`;
+
+    let broccoli6 = await obtainInstruction("broccoli", 6);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli6}</li>`;
+
+    let broccoli7 = await obtainInstruction("broccoli", 7);
+    document.querySelector("#broccoli").innerHTML += `<li>${broccoli7}</li>`;
+    document.getElementById("broccoliImg").removeAttribute("hidden");
+
+    let broccoli8 = await obtainInstruction("broccoli", 8);
+    document.querySelector("#broccoli").innerHTML += `<li>'${broccoli8}'</li>`;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+makeBroccoli();
 
 // Bonus 2 - Promise all
-// ...
\ No newline at end of file
+// ...
+
+Promise.all([
+  obtainInstruction("brusselsSprouts", 0),
```

Comment:
And here, you could've passed each obtainInstruction('food', step) to a different variable and then used them here. It would make your code a little bit more readable!